omim2obo/main.py

```diff
@@ -164,7 +164,7 @@ def omim2obo(use_cache: bool = False):
     # - Non-OMIM triples
     graph.add((URIRef('http://purl.obolibrary.org/obo/mondo/omim.owl'), RDF.type, OWL.Ontology))
     graph.add((URIRef(oboInOwl.hasSynonymType), RDF.type, OWL.AnnotationProperty))
-    graph.add((URIRef(MONDONS.omim_included), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(MONDONS.includedEntryInOMIM), RDF.type, OWL.AnnotationProperty))
```

Comment:
## Rename to `omimIncluded` instead
[Discussed w/ Nico today](https://docs.google.com/document/d/1zsfl5rkxdAhCk9xFX6oBDSa5UAjncBsloFuKU9Yt4Lo/edit?disco=AAABacqdWMQ). These are not the same.We need to keep both. 

- [ ] @joeflack4 Rename `omim_included` to `omimIncluded`
- [ ] @joeflack4 Don't remove `includedEntryInOMIM`